src/include/migraphx/generate.hpp

```diff
@@ -142,7 +142,7 @@ MIGRAPHX_EXPORT argument fill_argument(shape s, double value = 0);
 
 MIGRAPHX_EXPORT argument generate_argument(shape s,
                                            unsigned long seed = 0,
-                                           random_mode m      = random_mode::legacy);
+                                           random_mode m      = random_mode::random);
```

Comment:
default should be legacy, and only use random when benchmarking because this mode doesnt work well for verification.

It would be nice if we could call the mode `for_perf` or `for_benchmark` to make it clearer the intention.

Reply:
Add a comment on line 142 about what you are doing.  Even changing the enum to "for_perf" doesn't help someone looking at this in the future.

Reply:
> this mode doesnt work well for verification. 

Was running into this issue. Reverted changes to to make random mode as default.